src/ape/managers/chain.py

```diff
@@ -359,9 +360,23 @@ def base_fee(self) -> int:
     def pending_timestamp(self) -> int:
         """
         The current epoch time of the chain, as an ``int``.
+        Set the timestamp to fast-forward time into the future.
```

Comment:
For some reason, if this supports traveling backwards in limited circumstances (say during the genesis block), this docstring might not be right